interval/bounds.go

```diff
@@ -22,6 +22,24 @@ type Bounds struct {
 	index int
 }
 
+// NewBounds create a new Bounds given start and stop values
+func NewBounds(start, stop values.Time) *Bounds {
```

Comment:
See comment above. I fear that this method may not work the way we want it to.

Reply:
I'm guessing you mean that creating a new bounds is akin to using this like in range? The actual use case is mainly for testing purposes. Do you think that's okay or do you still want me to remove it?